src/config/routes.ts

```diff
@@ -15,6 +16,11 @@ const routes: Array<Route> = [
     label: 'Landing Page',
     component: LandingPage,
   },
+  {
+    path: '/CreateEventPage',
```

Comment:
This will become the page's path in the URL, so it should be written it all-lower-case with words separated by hyphens, i.e. `/create-event-page`. We'd probably want to drop the "page" suffix in this case though, so just `/create-event` should be enough.